Makefile

```diff
@@ -164,9 +173,10 @@ upgrade-dcos: check-terraform
 
 .PHONY: uninstall
 uninstall: check-dcos
-	$(DCOS_CMD) package uninstall --yes marathon-lb; \
-	$(DCOS_CMD) package uninstall --yes kubernetes; \
-	$(DCOS_CMD) marathon app remove kubeapi-proxy
+	$(DCOS_CMD) marathon app remove kubeapi-proxy || true
```

Comment:
Why `|| true`?

Reply:
+ 1

Reply:
Because the above will automatically stop the process even if the kubeapi-proxy is removed or doesnt exists, it exists with status 1.